package.json

```diff
@@ -26,19 +26,30 @@
     "start:server": "pm2 start ecosystem.config.js --env production",
     "build": "tsc",
     "build:watch": "tsc -w",
+    "test": "NODE_OPTIONS=--experimental-vm-modules jest",
     "update-data-sources": "node scripts/update-data-sources.js"
   },
   "prettier": {
     "singleQuote": false,
     "trailingComma": "all",
     "arrowParens": "avoid"
   },
+  "jest": {
+    "testRegex": "/tests/.*\\.test\\.js$",
+    "collectCoverageFrom": [
+      "build/**"
+    ],
+    "coverageProvider": "v8",
+    "transform": {}
+  },
   "devDependencies": {
     "@types/body-parser": "^1.19.0",
     "@types/express": "^4.17.11",
+    "@types/jest": "^26.0.20",
     "@types/node": "^14.14.25",
     "@types/node-fetch": "^2.5.8",
     "@types/split2": "^2.1.6",
+    "jest": "^26.6.3",
```

Comment:
Not sure if want to go with jest or not. 
Pros:
- ESM support
- Code coverage ((Had a hard time with mocha, jasmine here and above)
- Pretty, fast, maintained
- `expect()` API is pretty much same as we use in ReSpec

Cons:
- Facebook.
- Large number of dependencies that we might not even use

Reply:
I'm ok to try it out.